plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestTimestamp.java

```diff
@@ -23,13 +23,16 @@
 import io.trino.spi.block.Block;
 import io.trino.spi.connector.ConnectorPageSource;
 import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.type.BigintType;
 import io.trino.spi.type.SqlTimestamp;
+import io.trino.spi.type.Type;
 import org.apache.hadoop.hive.serde2.objectinspector.ObjectInspector;
 import org.apache.hadoop.mapred.JobConf;
 import org.apache.parquet.hadoop.metadata.CompressionCodecName;
 import org.apache.parquet.schema.MessageType;
 import org.testng.annotations.Test;
 
+import java.io.IOException;
```

Comment:
Squash the test changes with the fix ("Support reading parquet int64 timestamp as bigint" commit)